File: serverH/server/HostelOperationImpl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace serverH
{
    class HostelOperationImpl:MarshalByRefObject, HostelOperation
    {
        public static List<Hostel> lstHostel = new List<Hostel>();

        public List<Hostel> getListOfHostel()
        {
            return lstHostel;
        }

        public List<Hostel> addNewHostel(Hostel item)
        {
            lstHostel.Add(item);
            return lstHostel;
        }
        public List<Hostel> delHostel(int index)
        {
            lstHostel.RemoveAt(index);
            return lstHostel;
        }
        public int getSumOfHostel()
        {
            int sum = 0;
            foreach (Hostel hostel in lstHostel)
                sum += hostel.getPrice();
            return sum;
        }
    }
}

File: Server/src/operation/HostelOperationImpl.java
package operation;

import types.Hostel;

import java.util.ArrayList;
import java.util.List;

public class HostelOperationImpl implements HostelOperation{
    static List<Hostel> lstHostes = new ArrayList<Hostel>();

    static{
        lstHostes.add(new Hostel("Хостел 1",11,2,2000));
        lstHostes.add(new Hostel("Хостел 2",23,3,3000));
        lstHostes.add(new Hostel("Хостел 3",41,1,1000));
    }

    @Override
    public List<Hostel> getListOfHostel() {
        return lstHostes;
    }

    @Override
    public List<Hostel> addNewHostel(Hostel hostel){
        lstHostes.add(hostel);
        return lstHostes;
    }
    @Override
    public List<Hostel> delHostel(int index)
    {
        lstHostes.remove(index);
        return lstHostes;
    }

    @Override
    public int getSumOfHostel(){

        int sum = 0;
        for(Hostel hostel: lstHostes)
            sum += hostel.getPrice();
        return sum;
    }
}
